### PR TITLE
[GITHUB-35] Give bigger control over server.properties

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
It's now possible to specify any server property using
`sansible_kafka_server_properties` dictionary.

```
sansible_kafka_server_properties:
  offsets.topic.replication.factor: 1
```

This change also REMOVES:
* a lot of previous defaults for `server.properties`.
* AWS autodiscovery scripts

fixes: #23
fixes: #29

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,24 +1,17 @@
-ANSIBLE_INSTALL_VERSION ?= 2.5.9
+ANSIBLE_INSTALL_VERSION ?= 2.6.7
```
Comment (lobsterdore) [inline Makefile]: Do we definitely need these changes to the Makefile? It's fine if so but we will need to push these updates to the other roles at some point.

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ansible==2.5.9
```
Comment (lobsterdore) [inline requirements.txt]: Can you remove this Ansible requirement? It was put here in error a week or so ago.

Comment (wojtek-oledzki) [reply requirements.txt]: removing ...

File: Makefile (hunk 1 above)
Comment (wojtek-oledzki) [reply Makefile]: if the changes are ok, I will send PR to scaffold repo

**Comment by lobsterdore** [comment]
LGTM 👍

**Pull request merged**